requirements.txt

```diff
@@ -2,7 +2,6 @@ arrow
 requests>=2.0,<3.0dev
 units
 pytz
-six
```

Comment:
oh good i just asked about dropping six i'll do that and see if it fixes the dep resolve on test pypi in a conda envt